app/code/Magento/Tinymce3/view/base/web/tinymce3Adapter.js

```diff
@@ -78,6 +78,8 @@ define([
             }
 
             tinyMCE3.init(this.getSettings(mode));
+            varienGlobalEvents.clearEventHandlers("open_browser_callback");
+            varienGlobalEvents.attachEventHandler("open_browser_callback", tinyMceEditors.get(this.id).openFileBrowser);
```

Comment:
Please take a look at static tests as they expect single apostrophes to be used.